port_ocean/ocean.py

```diff
@@ -68,6 +71,26 @@ def __init__(
             integration_class(ocean) if integration_class else BaseIntegration(ocean)
         )
 
+    async def calculate_next_resync(self, now: datetime.datetime) -> float | None:
+        if self.config.runtime != "Saas" and not self.config.scheduled_resync_interval:
+            # There is no scheduled resync outside of Saas runtime or if not configured
+            return None
+
+        interval = self.config.scheduled_resync_interval
+        next_resync = None
+        if self.config.runtime == "Saas":
+            integration = await self.port_client.get_current_integration()
+            interval_str = (
+                integration.get("spec", {})
+                .get("appSpec", {})
+                .get("scheduledResyncInterval")
+            )
+            interval = convert_time_to_minutes(interval_str)
+
+        next_resync_date = now + datetime.timedelta(minutes=float(interval or 0))
+        next_resync = next_resync_date.now(datetime.timezone.utc).timestamp()
```

Comment:
lets leave it in seconds and make the UI show it in minutes if we want

Reply:
WDYM?
I changed it to `timestamp`  to have better control over how we show it, and changed it to UTC so we won't have any timezone issues